lib/modules/manager/bazel/extract.ts

```diff
@@ -73,13 +73,16 @@ const lexer = moo.states({
     },
     def: {
       match: new RegExp(
-        [
-          'container_pull',
-          'http_archive',
-          'http_file',
-          'go_repository',
-          'git_repository',
-        ].join('|')
+        '(?:' +
+          [
+            'container_pull',
+            'http_archive',
+            'http_file',
+            'go_repository',
+            'git_repository',
+            'maybe',
+          ].join('|') +
+          ')\\s*\\('
```

Comment:
Parens should be handled somewhere else

```suggestion
        [
          'container_pull',
          'http_archive',
          'http_file',
          'go_repository',
          'git_repository',
          'maybe',
        ].join('|')
```